src/main/java/com/launchdarkly/eventsource/EventSource.java

```diff
@@ -28,7 +28,7 @@
 
   private static final long DEFAULT_RECONNECT_TIME_MS = 1000;
   static final long MAX_RECONNECT_TIME_MS = 30000;
-  private final URI uri;
+  private URI uri;
```

Comment:
Thread safety is going to be a concern here. This should probably be `volatile`.